span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/validation/SpanProcessingConfigRequestValidator.java

```diff
@@ -96,6 +101,19 @@ private void validateRelationalSpanFilter(SpanFilter filter) {
     }
   }
 
+  private void validateUnarySpanFilter(SpanFilter filter) {
+    validateNonDefaultPresenceOrThrow(
+        filter.getUnarySpanFilter(), UnarySpanFilterExpression.OPERATOR_FIELD_NUMBER);
+
+    final LeftOperandCase leftOperandCase = filter.getUnarySpanFilter().getLeftOperandCase();
```

Comment:
I think I'm missing something. You're saying field must be set on 106, then immediately saying if field is set it's also an error?

Reply:
Sorry my mistake here - I saw the word FIELD and thought it was checking for the field.. field. But it's checking for the operator field.

Given this though, recommend removing this from the proto rather than relying on validation since it was just added and not yet in use.